_includes/footer.html

```diff
@@ -1,3 +1,4 @@
 <footer id="footer">
-    <p class="small">© Copyright {{ site.time | date: '%Y' }} {{ site.author }}</p>
+    <p class="small">© Copyright {{ site.time | date: '%Y' }} {{ site.author }} | <a href="https://github.com/rcos/rcos.github.io">Github</a></p>
```

Comment:
might be good to also link to rcos.io since it's the main place to see what rcos is up to